src/entity/learning-object/learning-object.ts

```diff
@@ -838,7 +838,8 @@ export namespace LearningObject {
 
   export namespace Material {
     export interface File {
-      id: string;
+      _id: string;
+      id: string; // Keeping for backward compat, may wanna remove?
```

Comment:
Hmm, unless someone says definitely to remove it, you could mark `id` as deprecated, or you could change its name to something like "source" like the way [learning-outcome model](https://github.com/Cyber4All/clark-service/blob/main/src/shared/types/learning-outcome.type.ts) is implemented.

Reply:
I believe it is safe to remove `id`. Please just double check where the File interface is being implemented/referenced throughout the client.

Reply:
done. the client builds, i believe this change should be okay to deploy =)